CodeEdit/Features/WindowCommands/FileCommands.swift

```diff
@@ -36,13 +36,8 @@ struct FileCommands: Commands {
         }
 
         CommandGroup(replacing: .saveItem) {
-            Button("Close Tab") {
```

Comment:
We should keep this item in the menu. Instead of putting the button in TabBarItemView, leave it here. We should be able to call a function that closes the active tab that is in the editor split that is currently in focus.

Reply:
Agreed, updated